scss/detail.scss

```diff
@@ -52,3 +53,15 @@
   width: 540px;
   border-bottom: 4px solid $earls-green;
 }
+
+.book-badge {
+  -ms-transform: rotate(15deg);
+  -webkit-transform: rotate(15deg);
+  bottom: 276px;
+  transform: rotate(15deg);
+  left: 210px;
+  position: absolute;
```

Comment:
It's always a good idea to z-index even when your browser sets the layer properly due to the fact that some browsers can have problem doing this, so it's better to specify a z-index instead.

Reply:
Done